gcn/notices/fermi/gbm/strict/alert.schema.json

```diff
@@ -0,0 +1,168 @@
+        "enum": ["on", "off", "triggered"]
+      },
+      "minItems": 14,
+      "maxItems": 14,
+      "description": "Triggered detectors"
+    },
+    "event_name": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "Name of the event, for example GRB170817A"
+    },
+    "hitl": {
+      "type": "boolean",
+      "default": false,
+      "description": "\"Human in the loop\".  When false, the notice was generated entirely automatically.  A value of true means that a human had an input, while the notice was being prepared.  The latter includes notices, which weren't altered by the human input, i.e. simply confirmed or approved by a human for further processing."
+    },
+    "id": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "type": "string"
+          },
+          {
+            "type": "number"
+          }
+        ]
+      },
+      "description": "Instrument-specific trigger ID, for example: 745121685, bn240812094"
+    },
+    "instrument": {
+      "const": "GBM",
+      "default": "GBM",
+      "description": "Name of the instrument reporting the event"
+    },
+    "latitude": {
+      "type": "number",
+      "description": "Geocentric latitude [deg]"
+    },
+    "light_curve_url": {
+      "type": "string",
+      "description": "URL of the quicklook light curve plot"
+    },
+    "longitude": {
+      "type": "number",
+      "description": "Geocentric longitude [deg]"
+    },
+    "messenger": {
+      "const": "EM",
+      "default": "EM",
+      "description": "Messenger of report; EM, GW or Neutrino"
+    },
+    "mission": {
+      "const": "Fermi",
+      "default": "Fermi",
+      "description": "Name of Mission or Telescope reporting the event"
+    },
+    "notice_type": {
+      "const": "FERMI_GBM_ALERT",
+      "description": "Notice title"
+    },
+    "rate_duration": {
+      "type": "number",
+      "description": "Duration in seconds of the ime interval used in the signal to noise ratio calculation"
+    },
+    "rate_energy_range": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "description": "Energy bounds in keV used in the signal to noise ratio calculation"
+    },
+    "rate_snr": {
+      "type": "number",
+      "description": "Signal to noise ratio"
+    },
+    "record_number": {
+      "type": "number",
+      "description": "Incremental number for notices related to the same trigger (ex: 1, 2, 3)"
+    },
+    "spectrum": {
+      "const": "energy",
+      "default": "energy",
+      "description": "high-energy or optical or radio observations, if not parsed, then default band is energy"
+    },
+    "trigger_algorithm_number": {
+      "type": "number",
+      "description": "The criterion number that caused the flight software to trigger. Not to be confused with flight software or any other version number."
```

Comment:
make it consistent with above definition.

Reply:
@shb46 reminder.

Reply:
@Vidushi-GitHub: Done. Thanks for the reminder.